metadata-helper/src/signatures.rs

```diff
@@ -58,8 +58,18 @@ async fn _index(
     let timer = SIGNATURES_SERVE_HIST.start_timer();
 
     let params = req.match_info();
-    let algo = params.get("ALGO").unwrap();
-    let digest = params.get("DIGEST").unwrap();
+    let sig = params.get("ALGO_DIGEST").unwrap();
```

Comment:
You have `signature` for the `{SIGNATURE}` portion of the URI.  Maybe use `algo_digest_string` or something for this variable, to avoid confusion with the `{SIGNATURE}` parameter?